lib/DI/ContainerConfigurator.php

```diff
@@ -104,6 +104,7 @@ function configure(ContainerBuilder $container) {
     $container->autowire(\MailPoet\Newsletter\AutomatedLatestContent::class)->setPublic(true);
     // WooCommerce
     $container->autowire(\MailPoet\WooCommerce\Helper::class)->setPublic(true);
+    $container->autowire(\MailPoet\WooCommerce\Subscription::class)->setPublic(true);
```

Comment:
It's used in integration tests. If we make it private, `SubscriptionTest` will say:
> Fetching the "MailPoet\WooCommerce\Subscription" private service or alias is deprecated since Symfony 3.4 and will fail in 4.0. Make it public instead.